src/game/Box.js

```diff
@@ -96,7 +103,8 @@ const Box = ({
         {boxValue == '.' || (
             <ImgContainer
                 src={'/static/pieceImage' + String(boxValue) + '.png'}
-                alt={String(clickable)}
+                alt={defineToolTip(badBox, goodBox, clickable)}
```

Comment:
You call this method two times at each render to have the same result. So you can do only one call at the top of the method and store the result in a constant.